app/components/emoji/emoji.tsx

```diff
@@ -130,7 +130,7 @@ const Emoji = (props: EmojiProps) => {
     return (
         <FastImage
             key={key}
-            style={[commonStyle, imageStyle, {width, height}]}
+            style={[(commonStyle as StyleProp<ImageStyle>), (imageStyle as StyleProp<ImageStyle>), {width, height}]}
```

Comment:
Instead of the `as`, probably this is better fixed at the `EmojiProps`, by using the correct `ImageStyle` there.

Reply:
I'll address this soon

Reply:
We are going to have the same problem a few lines above where the Image component is being used, so I needed to pick one or the other

Reply:
I see the issue... maybe we can use an intersection type as we did with the emojiProps? That intersection implementation I think is not the best (that was my first iteration trying to do it :P ), so you should probably use the one I recently created in the webapp:

```
export type Intersection<T1, T2> =
Omit<Omit<T1&T2, keyof(Omit<T1, keyof(T2)>)>, keyof(Omit<T2, keyof(T1)>)>;
```